### DISCUSSION TITLE
common css-var default names discussion

### DISCUSSION THREAD

**Discussion by xxholly32** [element-plus/element-plus#16709] [RFC]
css 默认的样式变量：

css-var names discussion

--- 

一些特殊后缀的：

Special suffix

+ --el-font-size-base
+ --el-border-radius-base
+ --el-z-index-normal
+ --el-common-component-size-default

没有后缀的：

whitout Special suffix

+ --el-color-primary
+ --el-bg-color
+ --el-border
+ --el-border-color
+ --el-fill-color
+ --el-box-shadow

--- 

那么可以看出来其实默认的后面加后缀就有不统一的情况出现，`base` `normal` `default` ，按照简介性原则用起来并不一定好理解，所以对于默认值是不是就是 不带后缀会比较合适呢？

So you can see that in fact, there is a non-uniform situation after the default suffix, `base` `normal` `default`  according to the principle of introduction is not always easy to understand, so for the default value is not with the suffix will be more appropriate?

**Comment by kooriookami** [comment]
样式命名这一块确实值得讨论，一些css变量目前是可有可无的状态。
